x/gov/client/utils/query_test.go

```diff
@@ -20,8 +20,16 @@ type TxSearchMock struct {
 	txs []tmtypes.Tx
 }
 
-func (mock TxSearchMock) TxSearch(query string, prove bool, page, perPage int, orderBy string) (*ctypes.ResultTxSearch, error) {
-	start, end := client.Paginate(len(mock.txs), page, perPage, 100)
+func (mock TxSearchMock) TxSearch(query string, prove bool, page, perPage *int, orderBy string) (*ctypes.ResultTxSearch, error) {
+	if page == nil {
+		*page = 0
+	}
+
+	if perPage == nil {
+		*perPage = 0
+	}
```

Comment:
@fedekunze any advice on what to do here or is this good?

Reply:
looks hacky. cc @sahith-narahari @aaronc?

Reply:
agreed. Not sure how to tackle this. it most likely something simple, im just blanking on it